cypress/support/commands.js

```diff
@@ -26,6 +26,10 @@
 
 const imgUrl = 'https://static.productionready.io/images/smiley-cyrus.jpg';
```

Comment:
The `imgUrl` constant is defined but not used anywhere in this file. If it's not used, it's better to remove it to avoid confusion.